src/protocolsupport/protocol/pipeline/version/v_pe/FromServerPacketDecoder.java

```diff
@@ -27,7 +28,7 @@
 		registry.register(Protocol.GAME, PEPacketId.Clientbound.PLAY_KICK, KickPacket.class);
 		registry.register(Protocol.GAME, PEPacketId.Clientbound.PLAY_START_GAME, LoginPacket.class);
 		registry.register(Protocol.GAME, PEPacketId.Dualbound.PLAY_CHAT, FromServerChatPacket.class);
-		registry.register(Protocol.GAME, PEPacketId.Clientbound.PLAY_RESPAWN, RespawnPacket.class);
```

Comment:
although im not sure why this was removed

Reply:
neither do I... @MrPowerGamerBR ?

Reply:
🤷‍♀️ 

I think I disabled it while testing and forgot to reenable.

Reply:
i put it back and test, seems fine?